RobotFramework_Testsuites/Config/robot_schema.json

```diff
@@ -1,16 +1,3 @@
-//  Copyright 2020-2022 Robert Bosch Car Multimedia GmbH
```

Comment:
Hi Son,
very good. I overlooked this file. 
Here comments are not allowed of course.
Thank you,
 Thomas